src/models/tools/tool-content-info.ts

```diff
@@ -28,6 +29,7 @@ export interface IToolContentInfo {
   defaultContent: (options?: IDefaultContentOptions) => ToolContentModelType;
   Component: ToolComponentType;
   toolTileClass: string;
+  icon?: FunctionComponent<SVGProps<SVGSVGElement>>;
```

Comment:
```suggestion
  Icon?: FunctionComponent<SVGProps<SVGSVGElement>>;
```
Since it's actually a component, I would capitalize it the way we've done with `Component`.

Reply:
done